Looks like there is an issue with FDS and OpenMPI 5.0 on macOS during the linking stage. Can you confirm it?
https://github.com/firemodels/fds/actions/runs/7243409341/job/19730178373
```
mpifort -m64 -check all -fp-stack-check -warn unused -O0 -auto -WB -g -traceback -fpe0 -nofltconsistency -stand:f18 -no-wrap-margin -fpp -DGITHASH_PP=\"-HEAD\" -DGITDATE_PP=\""Mon Dec 18 03:57:04 2023 +0000\"" -DBUILDDATE_PP=\""Dec 18, 2023  05:21:59\"" -DCOMPVER_PP=\""Intel ifort 2021.10.0"\" -DWITH_MKL -I/opt/intel/oneapi/mkl/2023.2.0/include -static-intel -o fds_ompi_intel_osx_db prec.o cons.o prop.o devc.o type.o data.o mesh.o func.o gsmv.o smvv.o rcal.o turb.o soot.o pois.o geom.o ccib.o radi.o part.o vege.o ctrl.o hvac.o mass.o wall.o fire.o velo.o pres.o init.o dump.o read.o divg.o main.o /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_intel_lp64.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_core.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_sequential.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_blacs_custom_lp64.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_intel_lp64.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_core.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_sequential.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_blacs_custom_lp64.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_intel_lp64.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_core.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_sequential.a /opt/intel/oneapi/mkl/2023.2.0/lib/libmkl_blacs_custom_lp64.a -lpthread -lm -ldl    
Undefined symbols for architecture x86_64:
  "_CFCopyTypeIDDescription", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_CFDataGetBytes", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_CFDataGetLength", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_CFDataGetTypeID", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_CFGetTypeID", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_CFNumberGetTypeID", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_CFNumberGetValue", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_CFRelease", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_CFStringGetCStringPtr", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_IOIteratorNext", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_IOObjectRelease", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_IORegistryEntryFromPath", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_IORegistryEntryGetChildIterator", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_IORegistryEntrySearchCFProperty", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "___CFStringMakeConstantString", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_kCFAllocatorDefault", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
  "_kIOMasterPortDefault", referenced from:
      _hwloc__look_darwin_cpukinds in libhwloc.a(topology-darwin.o)
ld: symbol(s) not found for architecture x86_64
make: *** [ompi_intel_osx_db] Error 1
```

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -26,9 +26,9 @@ permissions:
 env:
   # update urls for oneapi packages according to
   # https://github.com/oneapi-src/oneapi-ci/blob/master/.github/workflows/build_all.yml
-  WINDOWS_BASEKIT_URL: https://registrationcenter-download.intel.com/akdlm/IRC_NAS/f96c71db-2c6c-45d9-8c1f-0348ef5885cf/w_BaseKit_p_2023.2.0.49396_offline.exe
+  WINDOWS_BASEKIT_URL: https://registrationcenter-download.intel.com/akdlm/IRC_NAS/62641e01-1e8d-4ace-91d6-ae03f7f8a71f/w_BaseKit_p_2024.0.0.49563_offline.exe
   WINDOWS_BASEKIT_COMPONENTS: intel.oneapi.win.mkl.devel
-  WINDOWS_HPCKIT_URL: https://registrationcenter-download.intel.com/akdlm/IRC_NAS/438527fc-7140-422c-a851-389f2791816b/w_HPCKit_p_2023.2.0.49441_offline.exe
+  WINDOWS_HPCKIT_URL: https://registrationcenter-download.intel.com/akdlm/IRC_NAS/5b36181e-4974-4733-91c7-0c10c54900a5/w_HPCKit_p_2024.0.0.49588_offline.exe
   WINDOWS_HPCKIT_COMPONENTS: intel.oneapi.win.ifort-compiler:intel.oneapi.win.mpi.devel
 
 
@@ -77,9 +77,9 @@ jobs:
     - name: specify oneapi version
       run: |
         (
-          echo compiler=2023.2.0
-          echo mkl=2023.2.0
-          echo mpi=2021.10.0
+          echo compiler=2024.0.0
+          echo mkl=2024.0.0
+          echo mpi=2021.11.0
         ) > oneapi_config.txt
     - name: build fds debug
       run: |

